### PR TITLE
Michhhuang/tri 67 connect backend metric api to frontend dashboard to display

### PR DESCRIPTION
**Pull request by 1michhu1**
done :D

### CODE REVIEW

File: backend/api/views.py [hunk 1]
```diff
@@ -11,14 +11,16 @@
 def hello_world(request):
     return Response({"message": "ByeByeBias :)"})
 
-@api_view(["GET"])
+@api_view(["POST"])
 def get_dashboard_data(request):
-    TEMP_TEST_FILE = "backend/api/transaction_triple_b.parquet"
+    # TEMP_TEST_FILE = "backend/api/transaction_triple_b.parquet"
+    print(request.data)
```
Comment (1zhaohel) [inline backend/api/views.py:17]: delete line

**Review by chloehylee — APPROVED** [review]
amazing work michelle! lgtm (looks GREAT to me)

**Review by 1zhaohel — APPROVED** [review]
nice!! looks good (ﾉ◕ヮ◕)ﾉ*:・ﾟ✧

File: backend/api/views.py (hunk 1 above)
Comment (1michhu1) [reply backend/api/views.py:17]: fixed ✨

**Pull request merged**